### PR TITLE
Fix render to texture mips

### PR DESCRIPTION
**Pull request by sebavan**
https://forum.babylonjs.com/t/render-to-mipmap-level/44540

fix visible in https://playground.babylonjs.com/#T9J1HL

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

File: packages/dev/core/src/Engines/thinEngine.ts [hunk 1]
```diff
@@ -1933,6 +1933,15 @@ export class ThinEngine {
                     rtWrapper.texture!._hardwareTexture?.underlyingResource,
                     lodLevel
                 );
+            } else if (webglRTWrapper._currentLOD != lodLevel) {
```
Comment (Popov72) [inline packages/dev/core/src/Engines/thinEngine.ts:1936]: nit: !==

:)

**Review by Popov72 — APPROVED** [review]

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14418/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14418/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14418/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14418/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14418/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14418/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14418/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14418/merge#BCU1XR#0

**Pull request merged**